airflow/utils/db.py

```diff
@@ -68,6 +69,17 @@
 
 log = logging.getLogger(__name__)
 
+REVISION_HEADS_MAP = {
+    "2.0.0": "e959f08ac86c",
+    "2.0.1": "82b7c48c147f",
+    "2.0.2": "2e42bb497a22",
+    "2.1.0": "a13f7613ad25",
+    "2.1.3": "97cdd93827b8",
+    "2.1.4": "ccde3e26fe78",
+    "2.2.0": "7b2661a43ba3",
```

Comment:
Should we include all releases here? If we pass `2.2.1`, users would get "Please provide valid Airflow versions that has migrations between them." right? I think it'd be worth adding them all so it "just works" from their perspective.

Reply:
There's no migration head for that version unless we should use the head for a previous release that has migration but I think that a user using this feature is more likely to understand the message. WDYT?

Reply:
I think we should add all versions. It's very easy to add another command which will filter out the versions that have the same migration head as the previous one if you want "miigration versions only".

And It also gives a great opportunity to print a "happy" message to the users when they run "get me the script to migrate from "2.2.1 to 2.2.2":

```
# Hey this is your migration script from 2.2.1, to 2.2.2, but guess what? 
#
# There is no migration needed as the database has not changed between those versions. You are done.
#
#  /\_/\
# (='_' )
# (, (") (")
#
```